recipe/meta.yaml

```
@@ -1,13 +1,15 @@
 {% set name = "ESMValCore" %}
-{% set version = "2.10.0rc1" %}
+{% set lower_name = "esmvalcore" %}
+{% set version = "2.11.0rc1" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name }}
+  lower name: {{ lower_name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d467ce3ddbce7b5cfbbaa8ef60f06abfc29978f6a4573c786e1a19e7552d5348
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ lower_name }}-{{ version }}.tar.gz
```

How about using
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name|lower }}-{{ version }}.tar.gz
```
instead of introducing a new variable `lower_name`?

---

Done here: 975c215.

And to help make the distinction between `name` and `name` clear we've also gone for this as well? 03b7ced